people_schema.md

```diff
@@ -0,0 +1,239 @@
+***Required fields in people schema for group members
```

Comment:
I think you need to bookend this with `***`

Reply:
Thanks.  There is a lot of legacy bs in here I am trying to clean out so that my demo's go more smoothly.   It was a nasty rebase and I may have screwed some thingsy up but fingers crossed....